draconic/interpreter.py

```diff
@@ -854,8 +948,8 @@ def _eval_lambda(self, node):
     # executions
     def _eval_call(self, node):
         func = self._eval(node.func)
-        args = tuple(self._eval(a) for a in node.args)
-        kwargs = dict(self._eval(k) for k in node.keywords)
+        args = tuple(self._starred_unwrap(node.args, check_len=False))
+        kwargs = dict(self._starred_keyword_unwrap(((k.arg, k.value) for k in node.keywords), check_len=False))
```

Comment:
Why is `check_len` turned off here?

Reply:
Could be breaking, and it's checked on function side rather than call side already.